serialization/json.md

```diff
@@ -1,26 +1,30 @@
 # JSON Serialization
 
-Ideally JSON will be very similar (or even the same) to JSON-LD.
+## Parsing JSON-LD as JSON
+This is a description of how to deserialize JSON-LD as a pure JSON format without any knowledge of RDF.
+On top-level, JSON-LD has two keys, "@context" and "@graph".
 
-The JSON serialization will consist of two things: A context file and an array of element objects (the "payload").
+### parsing "@context"
+The context is a list of a string and an object. You can ignore the string.
+The object consists of key-value pairs that allow the shortening of IDs and which we will call "namespace map" in the following.
+For deserialization purposes, follow this process:
+- For every string that is an ID (that includes values of the keys "spdxId" and "@id", 
+  as well as all strings where you would expect objects according to the SPDX-3 model),
```

Comment:
> as well as all strings where you would expect objects according to the SPDX-3 model

Any hints on how to recognize these?  This may be one of the more confusing points.